includes/payments/class-edd-payment.php

```diff
@@ -2828,12 +2814,8 @@ private function maybe_alter_stats( $alter_store_earnings, $alter_customer_value
 		if ( ! empty( $this->customer_id ) ) {
 			$customer = new EDD_Customer( $this->customer_id );
 
-			if ( true === $alter_customer_value ) {
-				$customer->decrease_value( $this->total );
-			}
-
-			if ( true === $alter_customer_purchase_count ) {
-				$customer->decrease_purchase_count();
+			if ( $alter_customer_value || $alter_customer_purchase_count ) {
```

Comment:
I like to use `! empty()` around variable checks like these.

Keep in mind also that this is changing from a strict `true ===` checks to a loose one.